app/graphql/mutations/create_route_location.rb

```diff
@@ -0,0 +1,25 @@
+module Mutations
+  class CreateRouteLocation < BaseMutation
+
+    field :route_location, Types::RouteLocationType, null: false
+    field :errors, [String], null: false
+
+    argument :location_id, Types::LocationType, Integer, required: true
+    argument :route_id, Types::RouteType, Integer, required: true
+
+    def resolve(route_id:, location_id:, route_location:)
```

Comment:
Do you need `route_location` passed in? If it's a create than it should not exist.

```suggestion
    def resolve(route_id:, location_id:, route_location:)
```

Reply:
removed route_location